### PR TITLE
Adds Simple GUI | Simple Backend

### PR DESCRIPTION
**Pull request by gabru-md**
It adds a very simple GUI written in GTK.
Currently a call is given using the `system()` command. 
I'll work on it to see if it can be made better.

Cheers!

### CODE REVIEW

**Comment by gabru-md** [comment]
livewall

Screenshot of the GUI proposed.

File: makefile [hunk 1]
```diff
@@ -12,5 +12,8 @@ install:
 uninstall:
 	rm -f '/usr/local/bin/livewall'
 
+gui:
```
Comment (xeon-zolt) [inline makefile:15]: :+1:

**Comment by xeon-zolt** [comment]
I still think why are we using mpv as dependency I think we should remove that too

**Comment by gabru-md** [comment]
We need to work on the `livewall.c` for that purpose.
Then only we can change `mpv`.

**Pull request merged**